elliptic-curve/src/arithmetic.rs

```diff
@@ -84,3 +84,21 @@ pub trait PrimeCurveArithmetic:
     /// Prime order elliptic curve group.
     type CurveGroup: group::prime::PrimeCurve<Affine = <Self as CurveArithmetic>::AffinePoint>;
 }
+
+/// Perform a batched conversion to affine representation on a sequence of projective points
+/// at an amortized cost that should be practically as efficient as a single conversion.
+/// Internally, implementors should rely upon `InvertBatch`.
+pub trait ToAffineBatch: CurveArithmetic {
+    /// Converts a batch of points in their projective representation into the affine ones.
+    /// /// This variation takes a const-generic array and thus does not require `alloc`.
+    fn to_affine_batch_array<const N: usize>(
+        points: &[Self::ProjectivePoint; N],
+    ) -> [Self::AffinePoint; N];
```

Comment:
FWIW I opened an issue on `group` to suggest a trait like this: https://github.com/zkcrypto/group/issues/50